src/evict/evict_conn.c

```diff
@@ -323,7 +329,10 @@ __wt_evict_destroy(WT_SESSION_IMPL *session)
 
 /*
  * __wt_evict_stats_update --
- *     Update the eviction statistics for return to the application.
+ *     This function is responsible for initializing eviction stats, ensuring they start with
+ *     initial values during the startup process. It should be called exactly once when initializing
+ *     eviction. Running it outside of startup will not cause functional failures, but it will reset
+ *     eviction-related stats.
  */
 void
 __wt_evict_stats_update(WT_SESSION_IMPL *session)
```

Comment:
It's great that you call out in the comment what happens if you run this multiple times. 

What's not clear (I think you imply it) is that other things can break if this is not called - which makes me wonder why this is exposed as a public method and not called internally from say `evict_create`? Maybe you can test these comments with the question "does the comment make it obvious why this needs to be public"

```suggestion
__wt_evict_init_stats(WT_SESSION_IMPL *session)
```

Reply:
Good point. It currently called in two places:[`__wt_conn_stat_init`](https://github.com/wiredtiger/wiredtiger/blob/5f5bc8e8f04c3325a6ce518908d65178e49fe648/src/conn/conn_stat.c#L77) and [`__wt_evict_create`](https://github.com/wiredtiger/wiredtiger/blob/5f5bc8e8f04c3325a6ce518908d65178e49fe648/src/evict/evict_conn.c#L287). 

I think (think) we can remove the call in `__wt_conn_stat_init` and make this private, but our connnection_open logic is **very** hairy so I'd want this in a separate PR.

Reply:
Fair, I think it'd be reasonable for the solution in this pr to be a TODO in this comment block saying that

Reply:
Updated im [b91a788](https://github.com/wiredtiger/wiredtiger/pull/11096/commits/b91a7887129f1d67c8c92c89cedae30fc04cb722)